src/playwright-runner.ts

```diff
@@ -419,56 +425,64 @@ async function runPlaywright(nodeBin, runCfg) {
   }
   procArgs.push(...suite.testMatch);
 
-  args = _.defaultsDeep(suite, args);
```

Comment:
🤞

Reply:
Ha, fair enough! Restored the implementation, just to be safe.